tests/test_cli.py

```diff
@@ -393,13 +395,15 @@ def my_resource(rtype, *args):
         result = runner.invoke(cli, ['print', 'myapp.yaml', '--region=myregion', '123', 'master-mind'],
                                catch_exceptions=False)
 
-    stdout, cfjson = result.output.split('\n', 1)
-    assert 'Generating Cloud Formation template.. OK' == stdout
-    awsjson = json.loads(cfjson)
+    output = io.StringIO(result.output)
```

Comment:
Hi @tuxlife, *StringIO* just reads the contents of *result.output* as a stream and is used by *json.loads* to parse as if it was done from a file, while *.split* splits *result.output* into two new strings, the first line with the console output plus the duplicated json output. Furthermore the module *io* is a standard Python 3.x module and its use doesn't create new dependencies nor affects performance at all.

Anyway if you prefer to keep consistency across all tests I will revert StringIO usage into split with no problem. Hopefully we'll have this fix merged as soon as possible ;-)